includes/razorpay-webhook.php

```diff
@@ -54,28 +54,45 @@ public function process()
                     write_log(['message' => $e->getMessage()]);
                     return;
                 }
-            }
 
-            switch ($data['event'])
-            {
-                case 'payment.authorized':
-                    return $this->paymentAuthorized($data);
+                switch ($data['event'])
```

Comment:
@ccrims0n It should have, but @captn3m0 and I decided that we can push it out together and disabling the webhooks feature.